File: Python/card.py
from payment import Payment

class Card(Payment):
    name = str;
    number = int;
    cvv = int;
    date = int;

    def __init__(self, id, number, name, cvv, date):
        super().__init__(id);
        self.name = name;
        self.cvv = cvv;
        self.date = date;
        self.number = number;
        

File: Java/Card.java
public class Card extends Payment {
    Integer number;
    Integer cvv;
    Integer date;
    String name;

    public Card(Integer id, Integer number, Integer cvv, Integer date, String name){
        super(id);
        this.name = name;
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}